lib/CUDD/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ set(SOURCES
 
 add_llvm_library(Cudd ${SOURCES} LINK_LIBS m)
 
-set_target_properties(Cudd PROPERTIES COMPILE_FLAGS "-Wno-format -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -O3 -w -DHAVE_IEEE_754 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8")
```

Comment:
We will need to keep these options to properly use cudd on a 64 bit machine. The default macros are used for 32-bit ones.